src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Schemas/ProductType.cs

```diff
@@ -2,49 +2,187 @@
+        ///                fulfillmentCenterId
+        ///                fulfillmentCenterName
+        ///                allowPreorder
+        ///                allowBackorder
+        ///            }
+        ///        }
+        ///        properties{ id name valueType value valueId }
+        ///    }
+        ///}
+        /// </example>
+        public ProductType(IMediator mediator)
         {
             Name = "Product";
             Description = "Products are the sellable goods in an e-commerce project.";
 
-            //this.AuthorizeWith(CatalogModule.Core.ModuleConstants.Security.Permissions.Read);
-
             Field(d => d.CatalogProduct.Id).Description("The unique ID of the product.");
+
+            Field(d => d.CatalogProduct.Code, nullable: false).Description("The product SKU.");
+
+            Field<StringGraphType>("catalogId", resolve: context => context.Source.CatalogProduct.CatalogId);
+
+            FieldAsync<CategoryType>("category", resolve: async context =>
+            {
+                var categoryId = context.Source.CatalogProduct.CategoryId;
+                var responce = await mediator.Send(new LoadCategoryQuery
+                {
+                    Id = categoryId,
+                    IncludeFields = context.SubFields.Values.GetAllNodesPaths()
+                });
+
+                return responce.Category;
+            });
+
             Field(d => d.CatalogProduct.Name, nullable: false).Description("The name of the product.");
+
+            Field<ListGraphType<DescriptionType>>("descriptions", resolve: context => context.Source.CatalogProduct.Reviews);
+
             Field(d => d.CatalogProduct.ProductType, nullable: true).Description("The type of product");
-            Field(d => d.CatalogProduct.Code, nullable: false).Description("The product SKU.");
-            Field(d => d.CatalogProduct.ImgSrc).Description("The product main image URL.");
+
+            Field<StringGraphType>(
+                "slug",
+                description: "Get slug for product.",
+                resolve: context =>
+                {
+                    var semanticUrl = context.Source.CatalogProduct?.SeoInfos?.FirstOrDefault()?.SemanticUrl;
+                    return semanticUrl;
+                });
+
+            Field<StringGraphType>(
+                "metaDescription",
+                description: "Get metaDescription for product.",
+                resolve: context =>
+                {
+                    var metaDescription = context.Source.CatalogProduct?.SeoInfos?.FirstOrDefault()?.MetaDescription;
+                    return metaDescription;
+                });
+
+            Field<StringGraphType>(
+                "metaKeywords",
+                description: "Get metaKeywords for product.",
+                resolve: context =>
+                {
+                    var metaKeywords = context.Source.CatalogProduct?.SeoInfos?.FirstOrDefault()?.MetaKeywords;
```

Comment:
You can't use `FirstOrDefault` to find proper SeoInfo object, there is exists special logic for this purpose that take into account multiple factors, you need to move it to the xApi. https://github.com/VirtoCommerce/vc-storefront/blob/master/VirtoCommerce.Storefront/Domain/Catalog/CatalogConverter.cs#L384